src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Removed the direct dependency on `Microsoft.Extensions.Configuration.Binder`.
```

Comment:
Is this needed?

Reply:
Define needed 😄 Agree it is kind of silly to mention. Checked with @alanwest he was good with it NOT being mentioned so I removed.